test/common/testlib.py

```diff
@@ -1082,6 +1082,8 @@ def allow_journal_messages(self, *patterns):
     def allow_hostkey_messages(self):
         self.allow_journal_messages('.*: .* host key for server is not known: .*',
                                     '.*: refusing to connect to unknown host: .*',
+                                    '.*: .* host key for server has changed to: .*',
+                                    '.*: host key for this server changed key type: .*',
```

Comment:
Would it be too coarse to just allow `.*host key.*server.*`?

Reply:
I'd say so...